processes/security_team_members.md

```diff
@@ -0,0 +1,63 @@
+# Node.js Security Team
+
+Node.js security teams are expected to keep all information that they have
+privileged access to by being on the team completely private to the team. This
+includes notifying anyone outside the team of issues that have not yet been
+disclosed publically, including the existence of issues, expectations of
+upcoming releases, and patching of any issues other than in the process of
+their work as a member of the security team.
+
+Membership on the security teams can be requested via an issue in the TSC repo,
+and must be approved by current team members.
+
+Members of the security teams should indicate that they accept the privacy
+policies by PRing their acceptance to this file.
+
+## Team that triages security reports
+
+- @bnoordhuis - **Ben Noordhuis**
+- @indutny - **Fedor Indutny**
+- @rvagg - **Rod Vagg**
+- @jasnell - **James M Snell**
+- @shigeki - **Shigeki Ohtsu**
+- @MylesBorins - **Myles Borins**
+
+List is from ["security" alias](https://github.com/nodejs/email/blob/master/iojs.org/aliases.json).
+
+## Team with access to security issues
+
+- @ChALkeR - **Сковорода Никита Андреевич**
+- @Fishrock123 - **Jeremiah Senkpiel**
+- @MylesBorins - **Myles Borins**
+- @Trott - **Rich Trott**
+- @addaleax - **Anna Henningsen**
+- @bnoordhuis - **Ben Noordhuis**
+- @cjihrig - **Colin Ihrig**
+- @dougwilson - **Douglas Wilson**
+- @ejratl - **Emily Ratliff**
+- @evanlucas - **Evan Lucas**
+- @evilpacket - **Adam Baldwin**
+- @grnd - **Danny Grander**
+- @indutny - **Fedor Indutny**
+- @jasnell - **James M Snell**
+- @jbergstroem - **Johan Bergström**
+- @joaocgreis - **João Reis**
+- @joshgav - **Josh Gavant**
+- @mhdawson - **Michael Dawson**
+- @mscdex - **Brian White**
+- @ofrobots - **Ali Ijaz Sheikh**
+- @rvagg - **Rod Vagg**
+- @saghul - **Saúl Ibarra Corretgé**
+- @sam-github - **Sam Roberts**
+- @shigeki - **Shigeki Ohtsu**
+- @targos - **Michaël Zasso**
+- @thefourtheye - **Sakthipriyan Vairamani**
+- @trevnorris - **Trevor Norris**
+
+List is from [nodejs/teams/security](https://github.com/orgs/nodejs/teams/security/members).
+
+## Team with access to private security patches
+
+See [nodejs-private/node-private](https://github.com/nodejs-private/node-private).
+
+TBD - why is this list not the same as the team with access to security issues?
```

Comment:
I think it should be the same even if it is not currently.

Reply:
I just compared the lists.

@ejratl, @grnd, @mscdex, @shigeki, and @trevnorris are on the "access to security issues" list. They are not in `nodejs-private/node-private`, but all have pending invitations (it's a separate GitHub org).

@hackygolucky and @mrhinkle are included in `nodejs-private/node-private` but not the security issues list. They are probably included in `nodejs-private` because of Foundation activity.

Reply:
Yes, @hackygolucky and @mrhinkle must retain access from the foundation side of things.

Reply:
@mcollina, you said you had access to issues, but not node-private, but that isn't what @cjihrig is seeing above. comments?

Reply:
@mcollina wasn't listed here, so I didn't check his name. I can confirm he is not a member of node-private.

Reply:
Interesting, would he have access to the security issues as a side-effect of being on the TSC? Its quite possible my list is incomplete, I literally took the list from https://github.com/orgs/nodejs/teams/security/members, the folks in at-nodejs/security, but its possible there are more people with access than that team.